tests/basic/pure_test.go

```diff
@@ -335,3 +338,116 @@ var _ = Describe("{BringUpLargePodsVerifyNoPanic}", func() {
 		AfterEachTest(contexts, testrailID, runID)
 	})
 })
+
+var _ = Describe("{ResizePVCToMaxLimit}", func() {
+
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/72657582
+
+	var (
+		namespaces          = make([]string, 0)
+		contexts            = make([]*scheduler.Context, 0)
+		volumeMap           = make(map[string][]*api.Volume)
+		minVolSizeIncrement = uint64(1 * units.TiB)
+	)
+
+	// getMaxVolSize gets the maximum volume size based on the given volType
+	getMaxVolSize := func(volType string) uint64 {
+		switch volType {
+		case "FACD":
+			return 40 * units.TiB
+		default:
```

Comment:
check the max limit of other types like normal vsphere and make that as default

Reply:
Done.